nss-crystal-cobras/BangazonSite
# Ac home page

# Description

Closes Ticket #6 

User can see 20 most recent products on home page. I've added more products (there are 22 total) to the database using migrations. Each item is a hyperlink which routes user to its respective Details page. Details page doesn't currently meet ticket requirements, but that is a different ticket (which I will do). Once that is finished, will need to re-route to a different Details page.

The README documentation has also been set up to be added to as each team member pushes up ticket fulfillments.


## Type of change


# Testing Instructions

git fetch --all
git checkout ac-home-page
In SSMS, delete BangazonSite database (trust me)
In Visual Studio, delete Migrations folder (if you have one)
Open package manager console in Visual Studio (Tools --> NuGet Package Manager --> Package Manager Console)
--> run "Add-Migration NewProducts"
--> run "Update-Database"
Save all files in project
Run Bangazon debugger
Home page should show 20 products (last product being "Magnets")
Clicking on a product should route you to that product's details page

Look over README file for typos/wording/forgotten info

# Checklist:
